src/shared/json-transform.h

```diff
@@ -1,5 +1,11 @@
 #include "bus-message.h"
 #include "json.h"
 
-int json_transform_message(sd_bus_message *m, JsonVariant **ret);
-int json_transform_variant(sd_bus_message *m, const char *contents, JsonVariant **ret);
+typedef enum JsonTransformFlags {
+        JSON_TRANSFORM_PLAIN = 1 << 0,
+        JSON_TRANSFORM_TYPE_DATA  = 1 << 1, /* Insert layer with variant type/data */
```

Comment:
I'd prefer to have the equal signs aligned. Looks nicer when there's more entries.